refaster-test-support/src/main/java/tech/picnic/errorprone/refaster/test/RefasterRuleCollection.java

```diff
@@ -173,6 +186,68 @@ private void reportIncorrectClassName(CompilationUnitTree tree, VisitorState sta
     }
   }
 
+  private void reportUnSortedElidedTypesAndStaticImports(
+      CompilationUnitTree compilationUnit, VisitorState state, SuggestedFix.Builder fixBuilder) {
+    new TreeScanner<@Nullable Void, @Nullable Void>() {
+      @Override
+      public @Nullable Void visitMethod(MethodTree tree, @Nullable Void unused) {
+        String methodName = tree.getName().toString();
+        if (ELIDED_TYPES_AND_STATIC_IMPORTS_METHOD_NAME.equals(methodName)) {
+          Optional<ReturnTree> returnTree =
+              tree.getBody().getStatements().stream()
+                  .filter(statement -> statement.getKind() == Kind.RETURN)
+                  .findFirst()
+                  .map(ReturnTree.class::cast);
+
+          returnTree
+              .map(ReturnTree::getExpression)
+              .map(MethodInvocationTree.class::cast)
+              .ifPresent(
+                  methodInvocationTree -> {
+                    List<? extends ExpressionTree> actualArgumentsOrdering =
+                        methodInvocationTree.getArguments();
+
+                    ImmutableList<? extends ExpressionTree> desiredArgumentOrdering =
+                        actualArgumentsOrdering.stream()
+                            .sorted(
+                                comparing(
+                                    arg -> getArgumentName(arg, state),
+                                    String.CASE_INSENSITIVE_ORDER))
+                            .collect(toImmutableList());
+
+                    if (!actualArgumentsOrdering.equals(desiredArgumentOrdering)) {
+                      String suggestion =
+                          desiredArgumentOrdering.stream()
+                              .map(state::getSourceForNode)
+                              .collect(
+                                  joining(
+                                      ", ",
+                                      String.format(
+                                          "%s(",
+                                          state.getSourceForNode(
+                                              methodInvocationTree.getMethodSelect())),
+                                      ")"));
+
+                      fixBuilder.merge(SuggestedFix.replace(methodInvocationTree, suggestion));
```

Comment:
Curious why running `./apply-error-prone-suggestions.sh` did not fix the *(Input|Otput) tests here 🤔

Reply:
So what we are doing in the `RefasterRuleCollection`, is reporting things "in" the output file, that is what we are also verifying in the `refaster-test-support/src/test/resources/tech/picnic/errorprone/refaster/test/NonSortedElidedTypesAndStaticImportsTestRulesTestOutput.java` file. 

So e.g. in the `MatchInWrongMethodRulesTest` input file we try to see what is wrong and we _report_ that in the `MatchInWrongMethodRulesTest` output file, while not doing anything with the content itself, we only add a warning as a comment. 

So in the `NonSortedElidedTypesAndStaticImportsTestRulesTestOutput` file we should not change the output to be correct, we only need a nice comment explaining what is wrong. Instead of 
```
/* ERROR: Unexpected token. */
```
(which also signals something is wrong and the code we add has likely a bug)
It should add something like:
```
/*
   *  ERROR: The entries of the `elidedTypesAndStaticImports` method are not sorted lexicographically. The method should look like this:
   *   public ImmutableSet<Object> elidedTypesAndStaticImports() {
   *     return ImmutableSet.of(
              Assertions.class,
.....  (a method with correct sorting)
*/
```

This thing is not a "regular" BugPattern. It tries to guide a developer when writing TestInput & TestOutput files by giving this warning :).

Reply:
Thanks for explanation 🙇 🙇